.pipelines/v2/templates/pipeline-ci-build.yml

```diff
@@ -36,12 +36,6 @@ parameters:
     default: false
 
 stages:
-  # Allow manual builds to skip pre-check
-  - ${{ if ne(variables['Build.Reason'], 'Manual') }}:
-    - stage: Precheck
-      jobs:
-        - template: job-ci-precheck.yml
-
```

Comment:
You need to remove lines 42-44 or the builds will all implode

Reply:
Good catch